core-bundle/contao/languages/en/tl_image_size.xlf

```diff
@@ -77,6 +77,12 @@
       <trans-unit id="tl_image_size.skipIfDimensionsMatch.1">
         <source>Note that no optimizations such as metadata stripping or JPEG compression will be applied in this case.</source>
       </trans-unit>
+      <trans-unit id="tl_image_size.imageQuality.0">
+        <source>Image quality</source>
+      </trans-unit>
+      <trans-unit id="tl_image_size.imageQuality.1">
+        <source>Optional image compression quality between 0 (lowest quality) and 100 (lossless). If left empty, the system-wide configuration applies.</source>
```

Comment:
Minor nitpick: with WebP and JPEG the compression is never lossless, even at 100% quality, is it?

Reply:
https://imagemagick.org/script/webp.php

Reply:
Yes, it's only lossless if enabled. Anyway my suggestion would be to just write "highest quality".

Reply:
I have adjusted this in a1ff03f9c1af080c94b1e15589f0940ad2b189f1.